makefile.mingw

```diff
@@ -25,7 +25,7 @@ LTM_LDFLAGS = $(LDFLAGS) -static-libgcc
 #Libraries to be created
 LIBMAIN_S =libtommath.a
 LIBMAIN_I =libtommath.dll.a
```

Comment:
so this name and the one above is okay?

Reply:
Yes, that's OK. Static libraries are normally not versioned, just as header files like tommath.h. We wouldn't make that "tommath2.h" either.  Only runtime-files should have their version number in the file-name, because two dll's need to co-exist eventually on the same system.